code/controllers/communications.dm

```diff
@@ -144,9 +144,8 @@ var/list/radiochannels = list(
 	"Almayer"		= PUB_FREQ,
 	"Command"		= COMM_FREQ,
 	"MedSci"		= MED_FREQ,
-	"Engi"			= ENG_FREQ,
+	"Logistics"		= SUP_FREQ,
```

Comment:
```suggestion
	"Logistics"		= LOG_FREQ,
```

Reply:
Did this at first, but pretty sure it breaks some stuff, which is why CM was using `SUP_FREQ` for Requistions. Will look into making the changeover soon.